provider/rancher.go

```diff
@@ -290,7 +290,10 @@ func listRancherEnvironments(client *rancher.RancherClient) []*rancher.Environme
 
 	var environmentList = []*rancher.Environment{}
 
-	environments, err := client.Environment.List(nil)
+	withoutPagination := &rancher.ListOpts{
```

Comment:
How about setting up a "const" `ListOpts` and reuse it?

(where "const" could possibly mean: declare a var and initialize it in an `init()` function.)